src/pages/scroll-feature.js

```diff
@@ -0,0 +1,12 @@
+import React from "react"
```

Comment:
Not necessary for approval but we should remove this page now, since it was just a staging site for developing this animation.